Packetery/Checkout/Model/Payment/MethodList.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Packetery\Checkout\Model\Payment;
+
+use Packetery\Checkout\Model\Carrier\ShippingRateCode;
+
+class MethodList
+{
+    /**
+     * @var \Packetery\Checkout\Model\Carrier\Imp\Packetery\Carrier
+     */
+    private $packetery;
+
+    /**
+     * @var \Packetery\Checkout\Model\Pricing\Service
+     */
+    private $pricingService;
+
+    /**
+     * @param \Packetery\Checkout\Model\Carrier\Imp\Packetery\Carrier $packetery
+     * @param \Packetery\Checkout\Model\Pricing\Service $pricingService
+     */
+    public function __construct(\Packetery\Checkout\Model\Carrier\Imp\Packetery\Carrier $packetery, \Packetery\Checkout\Model\Pricing\Service $pricingService) {
+        $this->packetery = $packetery;
+        $this->pricingService = $pricingService;
+    }
+
+    /**
+     * @param \Magento\Payment\Model\MethodList $subject
+     * @param $availableMethods
+     * @param \Magento\Quote\Model\Quote|null $quote
+     * @return mixed|null
+     */
+    public function afterGetAvailableMethods(
+        \Magento\Payment\Model\MethodList $subject,
```

Comment:
nepoužitý parametr

Reply:
To tam předává Magento. S tím nic neudělám. Signaturu měnit nemužu.